network/p2p/test_utils.go

```diff
@@ -145,21 +161,32 @@ func (ln *LocalNet) NewTestP2pNetwork(ctx context.Context, nodeIndex int, keys t
 	//dutyStore := dutystore.New()
 	//signatureVerifier := signatureverifier.NewSignatureVerifier(nodeStorage)
 
-	cfg := NewNetConfig(keys, format.OperatorID(operatorPubkey), ln.Bootnode, testing.RandomTCPPort(12001, 12999), ln.udpRand.Next(13001, 13999), options.Nodes)
+	cfg := NewNetConfig(keys, format.OperatorID(operatorPubkey), ln.Bootnode, ssv_testing.RandomTCPPort(12001, 12999), ln.udpRand.Next(13001, 13999), options.Nodes)
 	cfg.Ctx = ctx
 	cfg.Subnets = "00000000000000000000020000000000" //PAY ATTENTION for future test scenarios which use more than one eth-validator we need to make this field dynamically changing
-	cfg.NodeStorage = mock.NodeStorage{
-		MockPrivateKeyHash:              hash,
-		RegisteredOperatorPublicKeyPEMs: []string{},
+	db, err := kv.NewInMemory(logger, basedb.Options{})
+	if err != nil {
+		return nil, err
 	}
+	ns, err := storage.NewNodeStorage(logger, db)
+	if err != nil {
+		return nil, err
+	}
+
+	shares := generateShares(t, ks, ns, networkconfig.TestNetwork)
+
+	cfg.NodeStorage = ns
 	cfg.Metrics = nil
 	// TODO: (Alan) decide if the code in this comment is needed, else remove
-	cfg.MessageValidator = nil //validation.New(
+	// cfg.MessageValidator = nil //validation.New(
 	//networkconfig.TestNetwork,
 	//nodeStorage.ValidatorStore(),
 	//dutyStore,
 	//signatureVerifier,
+	//validation.WithSelfAccept(selfPeerID, true),
 	//)
+	ctrl := gomock.NewController(t)
+	cfg.MessageValidator = newMockMessageValidator(ctrl, networkconfig.TestNetwork, ks, shares)
```

Comment:
@nkryuchkov I added validator storage with real shares from `spectestingutils.TestKeySet`. Does it look fine to you?

Reply:
@pavelkrolevets real shares are fine for me